src/pages/settings/OrganizationInformations.tsx

```diff
@@ -0,0 +1,350 @@
+  EditOrganizationInformationsDialog,
+  EditOrganizationInformationsDialogRef,
+} from '~/components/settings/EditOrganizationInformationsDialog'
+import CountryCodes from '~/public/countryCode.json'
+
+gql`
+  fragment OrganizationInformations on Organization {
+    id
+    logoUrl
+    name
+    legalName
+    legalNumber
+    email
+    addressLine1
+    addressLine2
+    zipcode
+    city
+    state
+    country
+  }
+
+  query getOrganizationInformations {
+    currentUser {
+      organizations {
+        ...OrganizationInformations
+        ...EditOrganizationInformationsDialog
+      }
+    }
+  }
+  ${EditOrganizationInformationsDialogFragmentDoc}
+`
+
+const OrganizationInformations = () => {
+  const { translate } = useInternationalization()
+  const editDialogRef = useRef<EditOrganizationInformationsDialogRef>(null)
+  const { data, loading, error } = useGetOrganizationInformationsQuery()
+  const organization = (data?.currentUser?.organizations || [])[0]
+
+  if (!!error) {
+    return (
+      <GenericPlaceholder
+        title={translate('text_629728388c4d2300e2d380d5')}
+        subtitle={translate('text_629728388c4d2300e2d380eb')}
+        buttonTitle={translate('text_629728388c4d2300e2d38110')}
+        buttonVariant="primary"
+        buttonAction={() => location.reload()}
+        image={<img src={EmojiError} alt="error-emoji" />}
+      />
+    )
+  }
+
+  return (
+    <Page>
+      <Title variant="headline">{translate('text_62ab2d0396dd6b0361614d2c')}</Title>
+      <Subtitle>{translate('text_62ab2d0396dd6b0361614d34')}</Subtitle>
+      <Head>
+        <Typography variant="subhead">{translate('text_62ab2d0396dd6b0361614d44')}</Typography>
+        <Button
+          variant="secondary"
+          disabled={!!loading}
+          onClick={editDialogRef?.current?.openDialog}
+        >
+          {translate('text_62ab2d0396dd6b0361614d3c')}
+        </Button>
+      </Head>
+      {!!loading ? (
+        <>
+          <Skeleton variant="connectorAvatar" size="medium" marginBottom={theme.spacing(4)} />
+          {[0, 1, 2, 3, 4, 5, 6, 7, 8, 9].map((skeletonLine) => (
+            <SkeletonLine key={`skeleton-${skeletonLine}`}>
+              <Skeleton variant="text" width={80} height={12} />
+              <Skeleton variant="text" width={240} height={12} />
+            </SkeletonLine>
+          ))}
+        </>
+      ) : (
+        <>
+          {organization.logoUrl ? (
+            <CompanyAvatar size="medium" variant="connector">
+              <img src={organization.logoUrl} alt={`${organization.name}'s logo`} />
+            </CompanyAvatar>
+          ) : (
+            <CompanyAvatar
+              size="medium"
+              variant="company"
+              identifier={organization.name || ''}
+              initials={(organization.name || '')
+                .split(' ')
+                .reduce((acc, n) => (acc = acc + n[0]), '')}
+            />
+          )}
+
+          <Table>
+            <tbody>
```

Comment:
Just for my knowledge... What's the point of using a table here ?
Is it really one ?

Reply:
I seemed more like a tabular data to me and also keeps the same width for the first column no matter the translation length. I could have used css grid too